components/clearbutton/metadata/clearbutton.yml

```diff
@@ -79,6 +79,16 @@ examples:
           </svg>
         </div>
       </button>
+  - id: clearbutton-quiet
+    name: Quiet
+    markup: |
+      <button class="spectrum-ClearButton spectrum-ClearButton--sizeM spectrum-ClearButton--quiet">
```

Comment:
Could we add the aria to prevent these buttons from being "empty", same as you did in `in-field button`. For each of the variants here.